Server export difference in v0.34.x

It looks like the exports were changed and maybe Server is not longer exported. Using this small example:

```
const Server = require('boardgame.io/server').Server;

const server = Server({
  games: [],
});

server.run(8000);
```
With:
* Node v13.1.0
* boardgame.io v0.34.2
* esm v3.2.25

Results in:

```
$ node -r esm server.js 
/home/nick/dev/tmp-boardgame.io/server.js:1
Error: Cannot find module 'boardgame.io/server'
Require stack:
- /home/nick/dev/tmp-boardgame.io/server.js
    at Object.<anonymous> (/home/nick/dev/tmp-boardgame.io/server.js:1)
    at Generator.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/nick/dev/tmp-boardgame.io/server.js' ]
}
```

It looks like it may not be exported anymore:

```
$ console.dir(require('boardgame.io));
{
  AI: [Function: AI],
  Client: [Function: Client],
  MCTSBot: [Function: MCTSBot],
  RandomBot: [Function: RandomBot],
  ReactClient: [Function: Client$1],
  ReactNativeClient: [Function: Client$2],
  TurnOrder: {
    DEFAULT: { first: [Function: first], next: [Function: next] },
    RESET: { first: [Function: first], next: [Function: next] },
    ONCE: { first: [Function: first], next: [Function: next] },
    CUSTOM: [Function: CUSTOM],
    CUSTOM_FROM: [Function: CUSTOM_FROM],
    SKIP: { first: [Function: first], next: [Function: next] }
  }
}
```

But I was able to find it at :
```
$ console.dir(require('boardgame.io/dist/server'));
{
  Firebase: [Function: Firebase],
  FlatFile: [Function: FlatFile],
  Mongo: [Function: Mongo],
  Server: [Function: Server]
}
```

So you can use `const Server = require('boardgame.io/dist/server').Server;` but that is weird. Advise on proper way to import it now?

--- a/scripts/clean.js
+++ b/scripts/clean.js
@@ -11,3 +11,4 @@ const subpackages = require('../subpackages');
 
 shell.rm('-rf', 'dist');
 shell.rm('-rf', subpackages);
+shell.rm('server.js');

--- a/scripts/proxy-dirs.js
+++ b/scripts/proxy-dirs.js
@@ -25,3 +25,8 @@ subpackages.forEach(name => {
   mkdirSync(dir);
   writeFileSync(`${dir}/package.json`, PackageJson(name));
 });
+
+writeFileSync(
+  path.resolve(__dirname, '../server.js'),
+  "module.exports = require('./dist/server');"
+);

